### ISSUE TITLE
FreeImage failed to build with latest libwebp update

### ISSUE THREAD

**Issue by lazyhamster** [Microsoft/vcpkg#2618]
After one of the latest updates FreeImage no longer is able to be built.
I guess problem is with latest libwebp update.
Build log contains the following errors:

> PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPNewInternal referenced in function "void * __stdcall Open(struct FreeImageIO *,void *,long)" (?Open@@YGPAXPAUFreeImageIO@@PAXJ@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxDelete referenced in function "void __stdcall Close(struct FreeImageIO *,void *,void *)" (?Close@@YGXPAUFreeImageIO@@PAX1@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxCreateInternal referenced in function "void * __stdcall Open(struct FreeImageIO *,void *,long)" (?Open@@YGPAXPAUFreeImageIO@@PAXJ@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxSetChunk referenced in function "long __stdcall Save(struct FreeImageIO *,struct FIBITMAP *,void *,int,int,void *)" (?Save@@YGJPAUFreeImageIO@@PAUFIBITMAP@@PAXHH2@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxGetChunk referenced in function "struct FIBITMAP * __stdcall Load(struct FreeImageIO *,void *,int,int,void *)" (?Load@@YGPAUFIBITMAP@@PAUFreeImageIO@@PAXHH1@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxSetImage referenced in function "long __stdcall Save(struct FreeImageIO *,struct FIBITMAP *,void *,int,int,void *)" (?Save@@YGJPAUFreeImageIO@@PAUFIBITMAP@@PAXHH2@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxGetFrame referenced in function "struct FIBITMAP * __stdcall Load(struct FreeImageIO *,void *,int,int,void *)" (?Load@@YGPAUFIBITMAP@@PAUFreeImageIO@@PAXHH1@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxGetFeatures referenced in function "struct FIBITMAP * __stdcall Load(struct FreeImageIO *,void *,int,int,void *)" (?Load@@YGPAUFIBITMAP@@PAUFreeImageIO@@PAXHH1@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxAssemble referenced in function "long __stdcall Save(struct FreeImageIO *,struct FIBITMAP *,void *,int,int,void *)" (?Save@@YGJPAUFreeImageIO@@PAUFIBITMAP@@PAXHH2@Z)
FreeImage.dll : fatal error LNK1120: 9 unresolved externals

FreeImage: 3.17.0-3
libwebp: 0.6.1

**Comment by lazyhamster** [comment]
Also for some reason libwebp build stopped producing dll. Previously there was webp.dll but not there are only lib files. Not sure if this is intended or not.

**Comment by rexdf** [comment]
My situation is that libwebp 0.6.1 build sucess while FreeImage 3.17.0-3 build failed.

```
CMakeFiles\FreeImage.dir\Source\Metadata\XTIFF.cpp.obj A:\vcpkg\installed\x86-windows\lib\zlib.lib A:\vcpkg\installed\x86-windows\lib\jpeg.lib A:\vcpkg\installed\x86-windows\lib\tiff.lib A:\vcpkg\installed\x86-windows\lib\lzma.lib A:\vcpkg\installed\x86-windows\lib\libpng16.lib A:\vcpkg\installed\x86-windows\lib\zlib.lib A:\vcpkg\installed\x86-windows\lib\openjp2.lib A:\vcpkg\installed\x86-windows\lib\webp.lib A:\vcpkg\installed\x86-windows\lib\jpegxr.lib A:\vcpkg\installed\x86-windows\lib\jxrglue.lib A:\vcpkg\installed\x86-windows\lib\raw.lib A:\vcpkg\installed\x86-windows\lib\IlmImf-2_2.lib A:\vcpkg\installed\x86-windows\lib\Imath-2_2.lib A:\vcpkg\installed\x86-windows\lib\IexMath-2_2.lib A:\vcpkg\installed\x86-windows\lib\IlmImfUtil-2_2.lib A:\vcpkg\installed\x86-windows\lib\Half.lib A:\vcpkg\installed\x86-windows\lib\Iex-2_2.lib A:\vcpkg\installed\x86-windows\lib\IlmThread-2_2.lib A:\vcpkg\installed\x86-windows\lib\jpeg.lib A:\vcpkg\installed\x86-windows\lib\tiff.lib A:\vcpkg\installed\x86-windows\lib\lzma.lib A:\vcpkg\installed\x86-windows\lib\libpng16.lib A:\vcpkg\installed\x86-windows\lib\webp.lib A:\vcpkg\installed\x86-windows\lib\jpegxr.lib A:\vcpkg\installed\x86-windows\lib\jxrglue.lib A:\vcpkg\installed\x86-windows\lib\raw.lib A:\vcpkg\installed\x86-windows\lib\IlmImf-2_2.lib A:\vcpkg\installed\x86-windows\lib\Imath-2_2.lib A:\vcpkg\installed\x86-windows\lib\IexMath-2_2.lib A:\vcpkg\installed\x86-windows\lib\IlmImfUtil-2_2.lib A:\vcpkg\installed\x86-windows\lib\Half.lib A:\vcpkg\installed\x86-windows\lib\Iex-2_2.lib A:\vcpkg\installed\x86-windows\lib\IlmThread-2_2.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib 
   Creating library FreeImage.lib and object FreeImage.exp
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPNewInternal referenced in function "void * __stdcall Open(struct FreeImageIO *,void *,long)" (?Open@@YGPAXPAUFreeImageIO@@PAXJ@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxDelete referenced in function "void __stdcall Close(struct FreeImageIO *,void *,void *)" (?Close@@YGXPAUFreeImageIO@@PAX1@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxCreateInternal referenced in function "void * __stdcall Open(struct FreeImageIO *,void *,long)" (?Open@@YGPAXPAUFreeImageIO@@PAXJ@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxSetChunk referenced in function "long __stdcall Save(struct FreeImageIO *,struct FIBITMAP *,void *,int,int,void *)" (?Save@@YGJPAUFreeImageIO@@PAUFIBITMAP@@PAXHH2@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxGetChunk referenced in function "struct FIBITMAP * __stdcall Load(struct FreeImageIO *,void *,int,int,void *)" (?Load@@YGPAUFIBITMAP@@PAUFreeImageIO@@PAXHH1@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxSetImage referenced in function "long __stdcall Save(struct FreeImageIO *,struct FIBITMAP *,void *,int,int,void *)" (?Save@@YGJPAUFreeImageIO@@PAUFIBITMAP@@PAXHH2@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxGetFrame referenced in function "struct FIBITMAP * __stdcall Load(struct FreeImageIO *,void *,int,int,void *)" (?Load@@YGPAUFIBITMAP@@PAUFreeImageIO@@PAXHH1@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxGetFeatures referenced in function "struct FIBITMAP * __stdcall Load(struct FreeImageIO *,void *,int,int,void *)" (?Load@@YGPAUFIBITMAP@@PAUFreeImageIO@@PAXHH1@Z)
PluginWebP.cpp.obj : error LNK2019: unresolved external symbol _WebPMuxAssemble referenced in function "long __stdcall Save(struct FreeImageIO *,struct FIBITMAP *,void *,int,int,void *)" (?Save@@YGJPAUFreeImageIO@@PAUFIBITMAP@@PAXHH2@Z)
FreeImage.dll : fatal error LNK1120: 9 unresolved externals
ninja: build stopped: subcommand failed.

```

**Comment by lazyhamster** [comment]
Fixed with 55b48e77d1742f807c27cdd6969a39cb54a8f8c2.
